lib/ohai/plugins/kernel.rb

```diff
@@ -116,7 +116,12 @@ def os_lookup(sys_type)
     so = shell_out("env lsmod")
     so.stdout.lines do |line|
       if line =~ /([a-zA-Z0-9\_]+)\s+(\d+)\s+(\d+)/
+        _status, version, _stderr = run_command(
```

Comment:
`run_command` will be deprecated in Ohai 9.0, please use `shell_out`.

Using `cat` to get system information is a bit unusual in Ohai. Does `"modinfo #{mod} -F version"` work?